studio/pages/api/auth/[ref]/users.ts

```diff
@@ -25,23 +25,69 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
 const handleGetAll = async (req: NextApiRequest, res: NextApiResponse) => {
   const headers = constructHeaders(req.headers)
-  const { keywords, limit, offset } = req.query
+  const { keywords, limit, offset, verified } = req.query
   const limitInt = tryParseInt(limit as string) || 10
   const offsetInt = tryParseInt(offset as string) || 0
   const hasValidKeywords = keywords && keywords != ''
-
+  const hasVerifiedValue = verified && verified != ''
   let queryCount = ''
   let queryUsers = ''
 
-  if (hasValidKeywords) {
+  if (hasValidKeywords && !hasVerifiedValue) {
     queryCount = SqlString.format('SELECT count(*) from auth.users WHERE email ilike ?;', [
       `%${keywords}%`,
     ])
     queryUsers = SqlString.format(
       'SELECT * from auth.users WHERE email ilike ? ORDER BY created_at DESC LIMIT ? OFFSET ?;',
       [`%${keywords}%`, limitInt, offsetInt]
     )
-  } else {
+  }
+
+  if (!hasValidKeywords && hasVerifiedValue) {
+    if (verified === 'verified') {
+      queryCount = SqlString.format(
+        'SELECT count(*) from auth.users WHERE (email_confirmed_at IS NOT NULL or phone_confirmed_at IS NOT NULL);'
+      )
+      queryUsers = SqlString.format(
+        'SELECT * from auth.users WHERE (email_confirmed_at IS NOT NULL or phone_confirmed_at IS NOT NULL) ORDER BY created_at DESC LIMIT ? OFFSET ?;',
+        [limitInt, offsetInt]
+      )
+    }
+    if (verified === 'unverified') {
+      queryCount = SqlString.format(
+        'SELECT count(*) from auth.users WHERE (email_confirmed_at IS NULL AND phone_confirmed_at IS NULL);'
+      )
+      queryUsers = SqlString.format(
+        'SELECT * from auth.users WHERE (email_confirmed_at IS NULL AND phone_confirmed_at IS NULL) ORDER BY created_at DESC LIMIT ? OFFSET ?;',
+        [limitInt, offsetInt]
+      )
+    }
+  }
+
+  if (hasValidKeywords && hasVerifiedValue) {
+    if (verified === 'verified') {
+      queryCount = SqlString.format(
+        'SELECT count(*)  from auth.users WHERE (email_confirmed_at IS NOT NULL or phone_confirmed_at IS NOT NULL) email ilike ?;',
```

Comment:
Missing `AND`

Reply:
@phamhieu - Just added a commit with the change.

Reply:
@phamhieu is this querying for you? I get a 404 on the users request.